deployments/gpu-operator/reconfigure-mig.go

```diff
@@ -63,8 +76,8 @@ func reconfigure(nodeName string, migConfigFile string, selectedMIGConfig string
 		i/p: migConfigFile, selectedMigConfig
 		o/p: true or false
 	*/
-	if err := assertConfigPresent(nvidiaMigPartedAlias, migConfigFile, selectedMIGConfig); err != nil {
-		setStateAndExit("failed", 1, currentLabels, hostGPUClientServices, withShutdownHostGPUClients, hostRootMount, nodeName)
+	if err := assertConfigPresent(migConfigProperties, selectedMIGConfig); err != nil {
```

Comment:
For all functions where `migConfigProperties` is now the first argument, we could consider hanging these functions off the type instead. For example:
```
func (c migConfig) assertConfigPresent(config string) error {
}
```
and then call it using `migConfigProperties.assertConfigPresent(selectedMIGConfig)` instead.

Reply:
done